### PR TITLE
added project euler problem 56,58,63,81,82,83

### PR DESCRIPTION
**Pull request by nehamsoni**
done all the changes according to guidelines
sorry for inconvenience i am little new to github 😉😁

### CODE REVIEW

**Comment by nehamsoni** [comment]
pls suggest the changes if needed 🙄

**Comment by cclauss** [comment]
Is this the replacement of #869?  If so, please edit the title of this PR to be the same as #869 so that the title of the PR tells us what it is trying to achieve.  🙏

**Comment by cclauss** [comment]
One thing that would be really helpful with these Euler solutions would be a comment that states the problem to be solved at the top of each file.

**Comment by nehamsoni** [comment]
> One thing that would be really helpful with these Euler solutions would be a comment at the top of each file that states the problem to be solved.

do you mean at top of each file i should specify
`# problem number <81> project Euler`

**Comment by cclauss** [comment]
https://projecteuler.net/problem=1

**Comment by nehamsoni** [comment]
done anything else...!!🙂

**Comment by nehamsoni** [comment]
added files for problem 82 and 83 aswell🙃

File: project_euler/problem_81/sol1.py [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+
+
+
+matrix=[]
+fi=open('input.txt','r')
+for line in fi:                         #.   matrix1 is used to calculate
+    line=line.split(',')                #.   d{} which counts repeated node's frequency
+    line=list(map(int, line))
+    matrix.append(line)
+fi.close()
```
Comment (cclauss) [inline project_euler/problem_81/sol1.py]: Replace seven lines with two:
```python
with open('input.txt') as in_file:
    matrix = [[int(x) for x in line.split(',')] for line in in_file]
```

Comment (nehamsoni) [reply project_euler/problem_81/sol1.py]: done.

File: project_euler/problem_81/sol1.py [hunk 2]
```diff
@@ -36,6 +36,7 @@ def dijkstra(initial,graph):
 
 with open('input.txt') as in_file:
     matrix = [[int(x) for x in line.split(',')] for line in in_file]
+in_file.close()
```
Comment (cclauss) [inline project_euler/problem_81/sol1.py]: __with open() as__ removes the need to call __.close()__.

Also one line was deleted in these changes that needs to be re-added.

Comment (nehamsoni) [reply project_euler/problem_81/sol1.py]: oh yess i hope its good now...😉

Comment (cclauss) [reply project_euler/problem_81/sol1.py]: Nice!  Thanks.

**Pull request closed without merging**